package.json

```diff
@@ -37,7 +37,7 @@
     "start": "yarn build-report --standalone && node ./lighthouse-cli/index.js",
     "jest": "node --experimental-vm-modules ./node_modules/jest/bin/jest.js",
     "test": "yarn diff:sample-json && yarn lint --quiet && yarn unit && yarn type-check",
-    "test-bundle": "yarn smoke --runner bundle -j=1 --retries=2",
+    "test-bundle": "yarn smoke --runner bundle --retries=2",
```

Comment:
bundle smokes really didn't like this change

https://github.com/GoogleChrome/lighthouse/runs/6219410614?check_suite_focus=true

Reply:
Yeah, doesn't seem to like it. Maybe you can try 4, like you previously suggested!

Reply:
I get the same thing locally, so I guess we have to run these tests serially. others seemed fine though.

Reply:
https://github.com/GoogleChrome/lighthouse/pull/9943#discussion_r343916850

Reply:
-j=1 was added at inception of the smokehouse bin: https://github.com/GoogleChrome/lighthouse/pull/9943/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R27

the failures are because we are clearing marky in every run of lighthouse, which is fine but the bundled smokehouse is the only one that actually runs lighthouse in the same process everytime

Reply:
> -j=1 was added at inception of the smokehouse bin: https://github.com/GoogleChrome/lighthouse/pull/9943/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R27

that's the same PR?

> the failures are because we are clearing marky in every run of lighthouse, which is fine but the bundled smokehouse is the only one that actually runs lighthouse in the same process everytime

Ah, I didn't look at the logs. That'll be two places it fails then :) I assume that's from the FR changes. Just say no to singletons

Reply:
hah, I misread and thought that linked to your other comment above....

Reply:
We could change `lighthouse-runners/bundle.js` to use a new process for each run (make a new file `bundle-entry.js` that just takes url, configJson, fraggleRock from argv) then we could remove this restriction. wdyt @brendankenny ?

Reply:
Yeah, it's kind of annoying in that we _should_ be able to use performance marks (and debug) in a way that doesn't run afoul of sharing a process, but you're right this is the odd one out, so it's not a huge deal.

- this might work with node [workers](https://nodejs.org/api/worker_threads.html)? Maybe no real benefit but it could be a single file solution (run the test unless `isMainThread`, in which case run itself as a worker)
- if we're really lazy, sharding works with the bundle runner as well, so can always shard these (and the existing smoke shard count can definitely be reduced when they run so fast on these runners :)